native/src/components/Background.tsx

```diff
@@ -0,0 +1,39 @@
+import React from "react";
+import {
+  ImageBackground,
+  StyleSheet,
+  KeyboardAvoidingView,
+} from "react-native";
+
+import { Theme } from "../Theme";
+
+export default function Background({ children }) {
+  return (
+    <ImageBackground
+      source={require("../../assets/empty.png")}
+      resizeMode="repeat"
+      style={styles.background}
+    >
+      <KeyboardAvoidingView style={styles.container} behavior="padding">
+        {children}
+      </KeyboardAvoidingView>
+    </ImageBackground>
```

Comment:
Mind walking through what this does? The empty image and `KeyboardAvoidingView` seem interesting, but it's not clear to me what they accomplish 😄

Reply:
`KeyboardAvoidingView` moves the contents when they keyboard is displayed, to avoid obfuscation. We probably don't need the `ImageBackground` for now, so will switch it to a view for the time being